core/node/rules/can_create_stream.go

```diff
@@ -242,6 +242,7 @@ func (ru *csParams) canCreateStream() ruleBuilderCS {
 			).
 			requireMembership(
 				inception.ChannelId,
+				inception.SpaceId,
 			).
 			requireChainAuth(ru.getChainAuthForMediaStream)
```

Comment:
Don’t you need to update this chain auth function?

can you make sure at least one of your tests is running against the entitled backend?

Reply:
added chainAuth for space